build_tools/packaging/linux/build_package.py

```diff
@@ -203,6 +206,11 @@ def generate_rules_file(pkg_info, deb_dir, config: PackageConfig):
     """
     print("Generate rules file")
     rules_file = Path(deb_dir) / "rules"
+    disable_dh_strip = (
+        None
+        if pkg_info.get("Disable_DH_STRIP") in (None, False, "False", "false")
+        else True
+    )
```

Comment:
nit: this could use a helper function to check for `in (None, False, "False", "false")`, since that code is repeated twice.

We have one such helper function in a different directory. Could look at this for ideas: https://github.com/ROCm/TheRock/blob/240ed2870072304217c6fba53be495a9802d7528/build_tools/github_actions/github_actions_utils.py#L134-L172